test/org/sagebionetworks/bridge/services/ConsentServiceImplTest.java

```diff
@@ -96,89 +89,81 @@ public void after() {
         try {
-            consentService.getConsentSignature(testUser.getUser(), testUser.getStudy());
+            consentService.getConsentSignature(testUser.getStudy(), testUser.getUser());
             fail("expected exception");
         } catch (EntityNotFoundException ex) {
-            thrownEx = ex;
         }
-        assertNotNull(thrownEx);
     }
 
     @Test
     public void canConsentWithSignatureImage() {
         // Consent and verify.
-        ConsentSignature researchConsent = ConsentSignature.create("Eggplant McTester", "1970-01-01",
+        ConsentSignature signature = ConsentSignature.create("Eggplant McTester", "1970-01-01",
                 TestConstants.DUMMY_IMAGE_DATA, "image/fake");
-        consentService.consentToResearch(testUser.getUser(), researchConsent, testUser.getStudy(), false);
-        assertTrue(consentService.hasUserConsentedToResearch(testUser.getUser(), testUser.getStudy()));
-        ConsentSignature returnedSig = consentService.getConsentSignature(testUser.getUser(), testUser.getStudy());
+        consentService.consentToResearch(testUser.getStudy(), testUser.getUser(), signature, false);
+        assertTrue(consentService.hasUserConsentedToResearch(testUser.getStudy(), testUser.getUser()));
+        
+        ConsentSignature returnedSig = consentService.getConsentSignature(testUser.getStudy(), testUser.getUser());
+        
         assertEquals("Eggplant McTester", returnedSig.getName());
         assertEquals("1970-01-01", returnedSig.getBirthdate());
         assertEquals(TestConstants.DUMMY_IMAGE_DATA, returnedSig.getImageData());
         assertEquals("image/fake", returnedSig.getImageMimeType());
 
         // Withdraw consent and verify.
-        consentService.withdrawConsent(testUser.getUser(), testUser.getStudy());
-        assertFalse(consentService.hasUserConsentedToResearch(testUser.getUser(), testUser.getStudy()));
-        EntityNotFoundException thrownEx = null;
+        consentService.withdrawConsent(testUser.getStudy(), testUser.getUser());
+        assertFalse(consentService.hasUserConsentedToResearch(testUser.getStudy(), testUser.getUser()));
         try {
-            consentService.getConsentSignature(testUser.getUser(), testUser.getStudy());
+            consentService.getConsentSignature(testUser.getStudy(), testUser.getUser());
             fail("expected exception");
         } catch (EntityNotFoundException ex) {
-            thrownEx = ex;
         }
-        assertNotNull(thrownEx);
     }
 
     @Test
     public void cannotConsentIfTooYoung() {
-        Study study = new DynamoStudy();
-        study.setIdentifier("api");
-        study.setName("Test Study");
-        study.setMinAgeOfConsent(18);
-
         LocalDate now = LocalDate.now();
         int year = now.getYear();
         int month = now.getMonthOfYear();
         int day = now.getDayOfMonth();
 
+        // The API study is set for 17 years... just go with that as it exists, it's easiest
```

Comment:
[MINOR] It looks like from the code change in ConsentBootstrapper and from the DDB tables, it's actually 18 years. Is this comment accurate?